docs/admin.rst

```diff
@@ -174,3 +174,24 @@ possession when they attempt to log in to SecureDrop.
 .. |Verify Google Authenticator works| image:: images/admin_enter_verification.png
 .. |Enable YubiKey| image:: images/admin_enable_yubikey.png
 .. |Verify YubiKey| image:: images/admin_verify_yubikey.png
+
+Managing `site-specific` updates on teams with multiple admins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Organizations with multiple admins should establish a protocol to communicate 
+any changes one admin makes to the `site-specifc` configuration file on the server. 
```

Comment:
`site-specific` needs to be in double backticks for it to be rendered as fixed width in ReST: \``site-specific``